src/powershell-native/Install-PowerShellRemoting.ps1

```diff
@@ -10,15 +10,14 @@
 #        plugin.
 #
 #####################################################################################################
+[CmdletBinding(DefaultParameterSetName = "NotByPath")]
 param
 (
-    [parameter(ParameterSetName = "ByPath")]
-    [ValidateNotNullOrEmpty()]
+    [parameter(Mandatory = $true, ParameterSetName = "ByPath")]
     [string]
     $PowerShellHome,
 
-    [parameter(ParameterSetName = "ByPath")]
-    [ValidateNotNullOrEmpty()]
+    [parameter(Mandatory = $true, ParameterSetName = "ByPath")]
     [string]
     $PowerShellVersion = "6.0.0-alpha.8"
```

Comment:
If the path is provided, we should get the powershell version using `powershell -v`and get rid of this parameter

Reply:
I agree.

Reply:
Sounds good. Will make the change.